crates/cli/src/app_builder.rs

```diff
@@ -162,99 +165,35 @@ pub trait AppBuilder: CommandBuilder {
 
                 assert!(public_inputs.len() <= Self::MAX_PUBLIC_INPUT_SIZE);
 
-                exec_create_proof(
+                exec_create_proof::<ExecutionArg, DefaultHostEnvBuilder>(
                     Self::NAME,
                     zkwasm_k,
                     wasm_binary,
                     phantom_functions,
                     &output_dir,
-                    public_inputs,
-                    private_inputs,
-                    context_in,
-                    context_out.clone(),
+                    &param_dir,
+                    ExecutionArg {
+                        public_inputs,
+                        private_inputs,
+                        context_inputs: context_in,
+                        context_outputs: context_out.clone(),
+                    }
                 )?;
 
                 write_context_output(&context_out.borrow(), context_out_path)?;
 
                 Ok(())
             }
             Some(("single-verify", sub_matches)) => {
-                let proof_path: PathBuf = Self::parse_proof_path_arg(&sub_matches);
-                let instance_path: PathBuf = Self::parse_single_instance_arg(&sub_matches);
+                let _proof_path: PathBuf = Self::parse_proof_path_arg(&sub_matches);
```

Comment:
remove them if they are no longer useful, as well as the related code in 'append_verify_single_proof_subcommand'.